Scripts/Runtime/Entities/ProxyDataStream/AbstractProxyDataStream.cs

```diff
@@ -0,0 +1,45 @@
+using Anvil.CSharp.Core;
+using Anvil.Unity.DOTS.Jobs;
+using System;
+using System.Reflection;
+using Unity.Jobs;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    public abstract class AbstractProxyDataStream : AbstractAnvilBase
+    {
+        internal static readonly BulkScheduleDelegate<AbstractProxyDataStream> CONSOLIDATE_FOR_FRAME_SCHEDULE_FUNCTION = BulkSchedulingUtil.CreateSchedulingDelegate<AbstractProxyDataStream>(nameof(ConsolidateForFrame), BindingFlags.Instance | BindingFlags.NonPublic);
+
+        public Type Type
+        {
+            get;
+        }
+
+        internal AccessController AccessController
+        {
+            get;
+        }
+
+        private readonly string m_DebugString;
+
+        protected AbstractProxyDataStream()
+        {
+            Type = GetType();
+            m_DebugString = $"{Type.Name[..^2]}<{Type.GenericTypeArguments[0].Name}>";
```

Comment:
Does this actually work? I thought you needed C#8
![image](https://user-images.githubusercontent.com/1426795/191782652-d44b49bb-866d-4e49-aab4-4a760433fe1e.png)

Reply:
Apparently we have access to C# 9.0 in Unity 2021.3 with a few caveats. 

https://docs.unity3d.com/Manual/CSharpCompiler.html

Everything is compiling and working.

Reply:
🤷 ok then. Fun!